yarn.lock

```diff
@@ -1312,13 +1312,20 @@
     tslint-microsoft-contrib "6.1.0"
     tslint-react "3.6.0"
 
-"@gooddata/typings@^2.18.0", "@gooddata/typings@^2.19.0":
+"@gooddata/typings@^2.18.0":
   version "2.19.0"
   resolved "https://registry.yarnpkg.com/@gooddata/typings/-/typings-2.19.0.tgz#59f6d825b3671bd17df13bf7020b2b856d94d923"
   integrity sha512-/r66t4g97WlO++dXPl17q5GF09l5LNrqbuPfKDOKtokUE3UiQYpUty/u975SkEg41TF1UlCw40Sikqo+AuanYw==
   dependencies:
     lodash "^4.17.11"
 
+"@gooddata/typings@^2.22.0":
```

Comment:
deduplicate this package

Reply:
Typings with version 2.18.0 from dependency of "@gooddata/mock-js@2.6.0" https://github.com/gooddata/gooddata-react-components/blob/develop/yarn.lock#L1274

Reply:
so can you please upgrade mock-js?

Reply:
updated